lib/perl/Genome/Model/Tools/Manta.pm

```diff
@@ -0,0 +1,15 @@
+package Genome::Model::Tools::Manta;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Manta {
+    is  => 'Command::Tree',
+    doc => 'Toolkit for executing the Manta structural variant call',
+};
+
+sub help_detail {
+    "These commands are used to execute Manta and related processes";
+}
```

Comment:
This module is missing a true value (`1;`) on the last line.

(In a more nit-picky comment, some of the other files are missing newlines at the end of them.)